fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -1732,6 +1732,12 @@ public void testJsonLdProfile() throws IOException {
         assertEquals("Should be two values!", 2, titles.findValues("@value").size());
     }
 
+    @Test
+    public void testEmptyPath() {
```

Comment:
This isn't an empty path. Maybe `testPathWithEmptySegment` or at least a comment indicating the eact meaning here?